clifford/_conformal_layout.py

```diff
@@ -66,7 +66,7 @@ def up(self, x: MultiVector) -> MultiVector:
                 new_val = np.zeros(self.gaDims)
                 new_val[:len(old_val)] = old_val
                 x = self.MultiVector(value=new_val)
-        except(AttributeError):
+        except (AttributeError):
```

Comment:
```suggestion
        except AttributeError:
```
I think this was left from python 2 compatibility